api/graphql/reservations/reservation_filtersets.py

```diff
@@ -37,6 +47,26 @@ class ReservationFilterSet(django_filters.FilterSet):
         field_name="user", queryset=User.objects.all()
     )
 
+    reservation_unit_name_fi = django_filters.CharFilter(
+        method="get_reservation_unit_name"
+    )
+    reservation_unit_name_en = django_filters.CharFilter(
+        method="get_reservation_unit_name"
+    )
+    reservation_unit_name_sv = django_filters.CharFilter(
+        method="get_reservation_unit_name"
+    )
+
+    unit = django_filters.ModelMultipleChoiceFilter(
+        method="get_unit", queryset=Unit.objects.all()
+    )
+
+    reservation_unit_type = django_filters.ModelMultipleChoiceFilter(
+        method="get_reservation_unit_type", queryset=ReservationUnitType.objects.all()
+    )
+
+    text_search = django_filters.CharFilter(method="get_text_search")
```

Comment:
I'm pondering this `text_search` naming. Looking inside the `get_text_search` function I see that there is only name related search done there.

Reply:
Yeah, I see your point. I guess we wanted this because we already had these "super searches" in other entities 🤔